frontend/review/src/pages/ReviewPage.js

```diff
@@ -118,7 +121,7 @@ export class ReviewPage extends Component {
           } else {
             this.setState({ data });
             if (type === "course") {
-              apiLive(data.code)
+              apiLive(data.code, semester)
                 .then(result => this.setState({ liveData: result }))
                 .catch(() => undefined);
```

Comment:
Why is this catch empty: shouldn't it put up some warnings about the course not being offered in the given semester?

Reply:
I think the user gets that feedback from not seeing the live data UI in PCR (stars, sections, status, etc). Don't think an explicit modal is necessary